### PR TITLE
Save and display comment history

### PR DESCRIPTION
**Pull request by ninaprabhu**

### CODE REVIEW

**Comment by ninaprabhu** [comment]
just wanted to note that I think this only "saves" comment history temporarily since the comments array will be re-initialized every time I restart the dev server

**Comment by Tetheta** [comment]
Starting review

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -22,29 +22,29 @@
...
   }
 
-
+  /* Print running list of comments, save history */
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = new Gson().toJson(messages);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = request.getParameter("text-container");
+    String[] array = text.split("\\s*,\\s*");
```
Comment (Tetheta) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: Interesting that you need to split on commas like this, does the text come to you in an array of "test, test2" kind of deal? If you're just splitting on spaces you might want to use str.split("\\s+") or some variation instead.

Also please add a comment above the regex explaining the intention. Regex is infamous for being hard to understand later (even simple uses like this).

Comment (ninaprabhu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: I wanted to store comma-delimited text inputs in separate Entities, definitely didn't make that clear and sorry about that! Will comment that in and make it clear in index.html as well

**Pull request merged**